Makefile

```diff
@@ -3,8 +3,7 @@ all                :; DAPP_LIBRARIES=' lib/dss-exec-lib/src/DssExecLib.sol:DssEx
                        dapp --use solc:0.6.12 build
 clean              :; dapp clean
                       # Usage example: make test match=SpellIsCast
-test               :; ./scripts/test-dssspell.sh match="$(match)" optimizer="$(optimizer)" block="$(block)"
-test-dev           :; ./scripts/test-dssspell.sh match="$(match)" optimizer="0" block="$(block)"
+test               :; ./scripts/test-dssspell.sh match="$(match)" optimizer="$(optimizer)" optimizer-runs="$(optimizer-runs)" block="$(block)"
 test-forge         :; ./scripts/test-dssspell-forge.sh match="$(match)" block="$(block)"
 estimate           :; ./scripts/estimate-deploy-gas.sh
 deploy             :; make && dapp create DssSpell | xargs ./scripts/verify.py DssSpell
```

Comment:
It looks like calling `make deploy` will deploy the contract with optimizations off. Is that where we want to be here?

Reply:
It seems that general consensus emerged so far is leaning towards testing and deploying spells without optimizations to speed up the process and reduce overhead.